tests/Fixtures/AccessorFinder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace JMS\Serializer\Tests\Fixtures;
+use JMS\Serializer\Annotation as Serializer;
+
+/** @Serializer\AccessorFinder() */
+final class AccessorFinder
+{
+    /**
+     * @Serializer\Type("integer")
+     */
+    private $a1_b = 1;
```

Comment:
can you also add one property that does not have getters/setters to see how it behaves if is missing

Reply:
can you also add one property that is marked as read-only to see how it behaves